cookbooks/bcpc-hadoop/recipes/resource_manager.rb

```diff
@@ -32,22 +32,54 @@
   not_if "hdfs dfs -test -d /var/log/hadoop-yarn", :user => "hdfs"
 end
 
+bash "create-hdfs-yarn-ats-log" do
+  code <<-EOH
+  hdfs dfs -mkdir -p /var/log/ats && hdfs dfs -chmod 1777 /var/log/ats && hdfs dfs -chown yarn:mapred /var/log/ats
```

Comment:
Why is this a bash resource instead of an hdfs_directory resource?

https://github.com/bloomberg/chef-bach/blob/master/cookbooks/bcpc-hadoop/libraries/hdfsdirectory.rb